projects/adventure-childrens-museum/deploy.yml

```diff
@@ -0,0 +1,102 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: Create a task, service, and ELB for this project
+Parameters:
+  Cluster:
+     Description: The name of the cluster where we should deploy
+     Type: String
+     Default: stage
+
+Resources:
+  # The task definition that says what container to run, what resources it needs, etc.
+  TaskDefinition:
+    Type: 'AWS::ECS::TaskDefinition'
+    Properties:
+      ContainerDefinitions:
+        - Name: museum-adventure
+          Image: !Sub
+            - 937163516619.dkr.ecr.us-west-2.amazonaws.com/museum-adventure:${cluster}
+            - { cluster: !Ref Cluster }
+          PortMappings:
+            - ContainerPort: 8080
+              HostPort: 8080
+          Essential: 'true'
+      Cpu: '256'
+      Memory: '512'
+      Family: !Ref AWS::StackName
+      NetworkMode: awsvpc
+      RequiresCompatibilities:
+         - FARGATE
+      ExecutionRoleArn:
+         Fn::ImportValue: !Sub "${Cluster}:ECSTaskRole"
+
+  # An ELB that we can expose our service on
+  ELB:
```

Comment:
Do we have to do a load balancer for every ecs service, or can we have a single application load balancer for all of them?

Reply:
We could build the alb in the fargate_cluster.yml and each new service will get a new mapping on the alb. We could likely do it subdomain based. 'projectname.mvpstudio.app' would forward to the production 'projectname' ecs service, 'projectname.stage.mvpstudio.app' would forward to the staging 'projectname' ecs service.